packages/run-protocol/test/amm/vpool-xyk-amm/test-liquidity.js

```diff
@@ -608,3 +608,62 @@ test('amm remove zero liquidity', async t => {
     `No liquidity removed`,
   );
 });
+
+test('amm bad add liquidity offer', async t => {
+  const centralLiquidityValue = 1_500_000_000n;
+  const secondaryLiquidityValue = 300_000_000n;
+
+  // Set up central token
```

Comment:
we have this boilerplate in many tests. I'd like to make a helper like `setupTokens('central', 'moola', 'sous')` that gives them consistent names. WDYT of that returning,
```
{
 central, // (value) => Amount
 centralBrand,
 centralIssuer,
  moola,
  …,
  sousIssuer,
}
```
They could of course be renamed in object destructuring assignment but the defaults would support some consistency.

Reply:
There are already a few helpers, and they're not as general as one might like. setupBasicMints in `run-protocol/test/swingsetTests/setup.js` only makes moola.  setupBasicMints.js in zoe unitTests is the right thing, but isn't exported cleanly.  

Would you object to a deep import of '@agoric/zoe/test/unitTests/setupBasicMints.js'?